test/FunctionalTests/Microsoft.OData.Core.Tests/ScenarioTests/Roundtrip/JsonLight/AsyncBatchRoundtripJsonLightTests.cs

```diff
@@ -242,5 +1216,17 @@ private void ClientReadAsyncBatchResponse(byte[] responsePayload, string batchCo
                 }
             }
         }
+
+        private byte[] ConvertStringToByteArray(string content)
+        {
+#if NETCOREAPP1_0
+            var stream = new MemoryStream(System.Text.Encoding.GetEncoding(0).GetBytes(content));

```

Comment:
Is this required for VS2017? I am using VS2015.

Reply:
I believe it is required for both--it's for the .NET Core version. I added this back in May when we did not move to VS2017 yet.